test/meson.build

```diff
@@ -1,15 +1,17 @@
-gtest_main_dep = dependency('gtest', main : true, fallback: ['gtest', 'gtest_main_dep'], required: false)
-gtest_dep = dependency('gtest', fallback: ['gtest', 'gtest_dep'], required: false)
+if not get_option('tests').disabled()
+  gtest_main_dep = dependency('gtest', main : true, fallback: ['gtest', 'gtest_main_dep'], required: true)
+  gtest_dep = dependency('gtest', fallback: ['gtest', 'gtest_dep'], required: true)
 
-test_inc = include_directories('.')
+  test_inc = include_directories('.')
 
-test_data_generator_sources = files('api/DataGenerator.cpp')
-test_base_encoder_sources = files('api/BaseEncoderTest.cpp')
+  test_data_generator_sources = files('api/DataGenerator.cpp')
+  test_base_encoder_sources = files('api/BaseEncoderTest.cpp')
 
-if gtest_dep.found()
-  subdir('api')
-  subdir('common')
-  subdir('decoder')
-  subdir('encoder')
-  subdir('processing')
-endif
+  if gtest_dep.found()
+    subdir('api')
+    subdir('common')
+    subdir('decoder')
+    subdir('encoder')
+    subdir('processing')
+  endif
+endif
```

Comment:
please use the linux file format. this line break does not seem to be a linux-formatted line break.

Reply:
This PR was prepared on a Linux machine. :)